packages/form-system/stories/01-overview.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Overview" />
+
+# Form System Overview
+
+This page should be used as a starting point when first using the Form System.
+It provides an overview of its essential building blocks and provides links to detailed explanations
+of most of its core concepts.
+
+## Building Blocks
+
+Our Form System is built from a set of very fundamental building blocks: `form control`s, `field`s and `fieldset`s.
+
+### Form Controls
+
+`Form control`s are the most fundamental building block of our Form System.
```

Comment:
What do you think about:

```suggestion
`Form control`s are the fundamental building blocks of our Form System.
```

Something can be fundamental or not. There is no half way fundamental.

Reply:
yep, block should be plural 👍

Reply:
Wrt fundamental. I think it can have various adjectives: 'most fundamental', 'very fundamental', 'quite fundamental' are all very commonly used.
In our particular case we have FormControl, Field and Input.
I would say FormControl is the most fundamental here, since it is shared by __al__l form components, whereas Field is shared by __many__ components.